ray-curator/pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     "torch",
     "transformers>=4.48.0",
     "pynvml==11.5.3",
+    "nemo_toolkit[asr]",
```

Comment:
Below under `project.optional-dependencies` we have `text`, `cuda12x`, `dev`, and `video` dependencies. Could you create a dependency for `audio` and move `nemo_toolkit[asr]` there?